content/docs/command-reference/import-url.md

```diff
@@ -62,7 +62,7 @@ DVC supports several types of (local or) remote locations (protocols):
 | `s3`     | Amazon S3                                           | `s3://mybucket/data`                       |
 | `azure`  | Microsoft Azure Blob Storage                        | `azure://my-container-name/path/to/data`   |
 | `gs`     | Google Cloud Storage                                | `gs://mybucket/data`                       |
-| `ssh`    | SSH server                                          | `ssh://user@example.com:/path/to/data`     |
+| `ssh`    | SSH server                                          | `ssh://user@example.com/path/to/data`      |
```

Comment:
same

Reply:
Auto-formatted. To align columns I guess. Looks "prettier" tehehe